mypyc/ops_exc.py

```diff
@@ -22,6 +22,13 @@
     format_str = 'raise_exception({args[0]}, {args[1]}); {dest} = 0',
     emit=simple_emit('PyErr_SetObject({args[0]}, {args[1]}); {dest} = 0;'))
 
+reraise_exception_op = custom_op(
+    arg_types=[],
+    result_type=bool_rprimitive,
+    error_kind=ERR_FALSE,
```

Comment:
Could you use `ERR_NEVER` here instead to simplify the generated code slightly (keeping everything else as is)?